packages/react-app/src/components/modals/TokenSelectorModal.jsx

```diff
@@ -54,21 +64,36 @@ export const TokenSelectorModal = ({ isOpen, onClose, onCustom }) => {
   // Callbacks
   const fetchTokenListWithBalance = useCallback(
     async tList => {
+      const tokenValueSortFn = ({ balance: balanceA }, { balance: balanceB }) =>
+        balanceB.sub(balanceA).gt(0) ? 1 : -1;
+
       const tokenListWithBalance = await Promise.all(
         tList.map(async token => ({
           ...token,
-          balance: await fetchTokenBalanceWithProvider(
-            ethersProvider,
-            token,
-            account,
-          ),
+          balance: NATIVE_CURRENCY_SYBMOLS.includes(token.symbol)
```

Comment:
Dont do this. There might be a token with the same symbol. Instead do a mode match maybe or an address === ADRESS_ZERO match.